server/src/main/java/org/spine3/server/aggregate/AggregatePartRepositoryLookup.java

```diff
@@ -62,19 +63,23 @@ private AggregatePartRepositoryLookup(BoundedContext boundedContext, Class<I> id
      *                               if not of the expected type, or
      *                               IDs are not of the expected type
      */
-    <A extends AggregatePart<I, S, ?>> AggregatePartRepository<I, A> find() {
-        final AggregateRepository<?, ?> repo = checkFound(boundedContext.getAggregateRepository(stateClass));
+    <A extends AggregatePart<I, S, ?, ?>> AggregatePartRepository<I, A, ?> find() {
+        final AggregateRepository<?, ?> repo =
+                checkFound(boundedContext.getAggregateRepository(stateClass));
 
         checkIsAggregatePartRepository(repo);
 
-        final AggregatePartRepository<I, A> result = checkIdClass((AggregatePartRepository<?, ?>)repo);
+        final AggregatePartRepository<I, A, ?> result =
+                checkIdClass((AggregatePartRepository<?, ?, ?>) repo);
         return result;
     }
 
     @SuppressWarnings("OptionalUsedAsFieldOrParameterType") // as this is the purpose of the method
-    private AggregateRepository<?, ?> checkFound(Optional<? extends AggregateRepository<?, ?>> rawRepo) {
+    private AggregateRepository<?, ?>
+    checkFound(Optional<? extends AggregateRepository<?, ?>> rawRepo) {
         if (!rawRepo.isPresent()) {
-            final String errMsg = String.format("Unable to find repository for the state class: %s", stateClass);
+            final String errMsg = String.format("Unable to find repository for the state class: %s",
```

Comment:
Use static import for `String.format()` and it would make the line shorter. Hence you'd avoid wrapping in this case.